### PR TITLE
Daos 6338 DAOS:Change DAOS license text #5

### PR DESCRIPTION
**Pull request by GitHuaKuang**
Updated mainly .java + some found missed files

### CODE REVIEW

File: src/client/java/daos-java/src/main/java/io/daos/BufferAllocator.java [hunk 1]
```diff
@@ -1,24 +1,7 @@
 /*
  * (C) Copyright 2018-2020 Intel Corporation.
```
Comment (cibervicho) [inline src/client/java/daos-java/src/main/java/io/daos/BufferAllocator.java:2]: 2018-2021

File: src/client/java/daos-java/src/main/java/io/daos/Constants.java [hunk 2]
```diff
@@ -1,24 +1,7 @@
 /*
  * (C) Copyright 2018-2020 Intel Corporation.
```
Comment (cibervicho) [inline src/client/java/daos-java/src/main/java/io/daos/Constants.java:2]: 2018-2021

File: src/client/java/daos-java/src/main/java/io/daos/DaosClient.java [hunk 3]
```diff
@@ -1,24 +1,7 @@
 /*
  * (C) Copyright 2018-2020 Intel Corporation.
```
Comment (cibervicho) [inline src/client/java/daos-java/src/main/java/io/daos/DaosClient.java:2]: 2018-2021

File: src/client/java/daos-java/src/main/java/io/daos/DaosContainer.java [hunk 4]
```diff
@@ -1,24 +1,7 @@
 /*
  * (C) Copyright 2018-2020 Intel Corporation.
```
Comment (cibervicho) [inline src/client/java/daos-java/src/main/java/io/daos/DaosContainer.java:2]: 2018-2021

File: src/client/java/daos-java/src/main/java/io/daos/DaosEventQueue.java [hunk 5]
```diff
@@ -1,24 +1,7 @@
 /*
  * (C) Copyright 2018-2020 Intel Corporation.
```
Comment (cibervicho) [inline src/client/java/daos-java/src/main/java/io/daos/DaosEventQueue.java:2]: 2018-2021

File: src/client/java/daos-java/src/main/java/io/daos/DaosIOException.java [hunk 6]
```diff
@@ -1,24 +1,7 @@
 /*
  * (C) Copyright 2018-2020 Intel Corporation.
```
Comment (cibervicho) [inline src/client/java/daos-java/src/main/java/io/daos/DaosIOException.java:2]: 2018-2021

File: src/client/java/hadoop-daos/src/test/java/io/daos/fs/hadoop/DaosConfigFileTest.java [hunk 7]
```diff
@@ -1,19 +1,5 @@
 /**
```
Comment (cibervicho) [inline src/client/java/hadoop-daos/src/test/java/io/daos/fs/hadoop/DaosConfigFileTest.java:1]: The copyright is missing

**Review by cibervicho — CHANGES_REQUESTED** [review]
Seems that non of the years in the format of '2018-2020' in all Java files were updated with '2018-2021'.
Also noticed that several java files don't even have the copyright message.

With respect to the python, header and C files, all look good to me.

File: src/client/java/daos-java/src/main/java/io/daos/BufferAllocator.java (hunk 1 above)
Comment (GitHuaKuang) [reply src/client/java/daos-java/src/main/java/io/daos/BufferAllocator.java:2]: Good catch - the current pre-commit doesn't handle .java, will try to update it.

File: src/client/java/hadoop-daos/src/test/java/io/daos/fs/hadoop/DaosConfigFileTest.java (hunk 7 above)
Comment (GitHuaKuang) [reply src/client/java/hadoop-daos/src/test/java/io/daos/fs/hadoop/DaosConfigFileTest.java:1]: OK, the original file doesn't have Copyright, this ticket does not intend to add that.

**Review by cibervicho — APPROVED** [review]

**Review by jolivier23 — APPROVED** [review]
I think the java files should have a copyright

**Comment by jolivier23** [comment]
@zjf2012 is the lack of copyright in some of the files intentional?

**Comment by zjf2012** [comment]
> @zjf2012 is the lack of copyright in some of the files intentional?

There are test files. We'd better add copyright. I'll create a ticket for that.

**Comment by johannlombardi** [comment]
Commit message is not following our standard. @GitHuaKuang please pay attention to this.

**Pull request merged**